tests/unit/test_messages_stream_cm.py

```diff
@@ -201,3 +201,15 @@ def test_pipereader_chunk_assembler(recording_printer, monkeypatch):
     msg1, msg2 = recording_printer.written_terminal_lines
     assert msg1.text == ":: ------abcde---"
     assert msg2.text == ":: otherline---"
+
+
+def test_no_fail_if_big_number_is_used(recording_printer):
```

Comment:
Hello! Thanks for the test. 

I'd suggest a more "unittesty" one: just start the _PipeReaderThread, stop it, and then assert that both pipes are closed. (the current approach has the problem that is testing the side-effect, which may change if we modify internals... for example, if we start using one pipe and forgot to close it, this will still pass...)

Thanks again!

Reply:
Should I add another unitary test, or replace the current one?

I think that it would be better to do both...

Reply:
Added the new test.